### PR TITLE
Add license and better documentation

### LINKED ISSUE #5: Update Repository Information
## README:

- Add an Installation Guide detailing the steps to set up the project.
Include a Usage Guide with example usage to help users understand how 
to work with the project.

- Add a section explaining the DAI organisation.

## Repository Description:

- Update the repository description to clearly state the project’s purpose and functionality.

## License:

- Choose an appropriate open-source license (e.g., MIT, Apache 2.0).
- Add the license file to the repository.

### PR DESCRIPTION
**Pull request by A2va**
Closes #5

### CODE REVIEW

**Comment by A2va** [comment]
Before merging, I need to update the task list

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+ MIT License
+
+ Copyright (c) 2022 Robin Forestier (Forestierr)
```
Comment (Forestierr) [inline LICENSE]: It shouldn't be 2024 ?

File: src/main/java/ch/heigvd/dai/Main.java [hunk 2]
```diff
@@ -1,3 +1,25 @@
+// MIT License
+//
+// Copyright (c) 2022 Robin Forestier (Forestierr)
```
Comment (Forestierr) [inline src/main/java/ch/heigvd/dai/Main.java]: Same here for 2022 -> 2024

File: src/main/java/ch/heigvd/dai/commands/Root.java [hunk 3]
```diff
@@ -1,3 +1,25 @@
+// MIT License
+//
+// Copyright (c) 2022 Robin Forestier (Forestierr)
```
Comment (Forestierr) [inline src/main/java/ch/heigvd/dai/commands/Root.java]: and here 2022 -> 2024

File: LICENSE (hunk 1 above)
Comment (A2va) [reply LICENSE]: Yeah, that was a copy paste error

**Review by Forestierr — APPROVED** [review]
All good for me.

**Pull request merged**